gulpfile.js

```diff
@@ -76,8 +77,10 @@ gulp.task('tests:sauce:start', function (done) {
     );
 
     tunnel.start(function (isCreated) {
+        isTunnelCreated = isCreated;
         if (!isCreated) {
-            done('Failed to create Sauce tunnel.');
+            done();
```

Comment:
actually now that i think of it, maybe we could do a console.log here like `Failed to create Sauce tunnel, skipping tests`